yarn.lock

```diff
@@ -11536,13 +11531,10 @@ react-scripts@^3.2.0:
   optionalDependencies:
     fsevents "2.0.7"
 
-react-side-effect@^1.1.0:
-  version "1.1.5"
-  resolved "https://registry.yarnpkg.com/react-side-effect/-/react-side-effect-1.1.5.tgz#f26059e50ed9c626d91d661b9f3c8bb38cd0ff2d"
-  integrity sha512-Z2ZJE4p/jIfvUpiUMRydEVpQRf2f8GMHczT6qLcARmX7QRb28JDBTpnM2g/i5y/p7ZDEXYGHWg0RbhikE+hJRw==
-  dependencies:
-    exenv "^1.2.1"
-    shallowequal "^1.0.1"
+react-side-effect@2.1.0, react-side-effect@^1.1.0:
+  version "2.1.0"
```

Comment:
If I am understanding it fine, it says that we will use the version `2.1.0` even when the client asks for version `1.1.0`.

It could have breaking changes (major version number has changed), is it acceptable? do you know if it is only being used in tests?

Reply:
I will check, this was supposed to be only for react-helmet, not for all other packages.

Reply:
It's only used by `react-helmet` which we use to update html document properties like title and meta tags.

Reply:
The breaking change is related to the React version used, so, we are fine:

![image](https://user-images.githubusercontent.com/1157864/68386643-3c9f6a00-013b-11ea-890c-a2581781e19d.png)